packages/office-ui-fabric-react/src/utilities/dateMath/DateMath.ts

```diff
@@ -198,4 +198,73 @@ function getDatePartHashValue(date: Date) {
   /* tslint:disable:no-bitwise */
   return date.getDate() + (date.getMonth() << 5) + (date.getFullYear() << 9);
   /* tslint:enable:no-bitwise */
-}
\ No newline at end of file
+}
+
+/**
+ * Returns the week numbers for each week in a month.
+ * Week numbers are 1 - 52 (53) in a year
+ */
+export function getWeekNumbersInMonth(weeks: any[], firstDayOfWeek: DayOfWeek, navigatedDate: Date) {
```

Comment:
>weeks: any[], [](start = 38, length = 13)

Hmm, this is kind of knowledge specific to the calendarDay so my suggestion is that you move this function to the CalendarDay.tsx file